plugins/node/opentelemetry-instrumentation-redis-4/src/instrumentation.ts

```diff
@@ -166,23 +166,30 @@ export class RedisInstrumentation extends InstrumentationBase<any> {
         this._diag.debug('Patching redis client');
         const redisClientPrototype = moduleExports?.default?.prototype;
 
-        if (isWrapped(redisClientPrototype?.multi)) {
-          this._unwrap(redisClientPrototype, 'multi');
+        // In some @redis/client versions 'multi' is a method. In later
+        // versions, as of https://github.com/redis/node-redis/pull/2324,
+        // 'MULTI' is a method and 'multi' is a property defined in the
+        // constructor that points to 'MULTI'.
```

Comment:
Thanks for adding the comment here to point out the reason. One suggestion (non-blocking): to me the wording in the issue that explained that it's not part of the prototype helped in understanding the changes in this PR, maybe we should also add it here :) 
```suggestion
        // In some @redis/client versions 'multi' is a method. In later
        // versions, as of https://github.com/redis/node-redis/pull/2324,
        // 'MULTI' is a method and 'multi' is a property defined in the
        // constructor that points to 'MULTI', and therefore it will not
        // be defined on the prototype.
```